src/utils/testsToStatusUpdate.test.ts

```diff
@@ -15,7 +15,7 @@ it("handles single test", () => {
     {
       "story--id": {
         "description": "Chromatic Visual Tests",
-        "status": "success",
```

Comment:
That's not a valid status value. `I think you're looking for `unknown`:
https://github.com/storybookjs/storybook/blob/2e2f7dd191c94bd07b856264c81023223d8e8ecf/code/lib/types/src/modules/api-stories.ts#L176